app_dart/lib/src/model/appengine/branch.dart

```diff
@@ -0,0 +1,66 @@
+// Copyright 2021 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:cocoon_service/src/model/appengine/key_converter.dart';
+import 'package:gcloud/db.dart';
+import 'package:github/github.dart';
+import 'package:json_annotation/json_annotation.dart';
+
+part 'branch.g.dart';
+
+@Kind(name: 'Branch', idType: IdType.String)
+class Branch extends Model<String> {
+  Branch({Key<String>? key, required int lastActivity}) {
+    parentKey = key?.parent;
+    id = key?.id;
+  }
+
+  /// The timestamp (in milliseconds since the Epoch) of the last time
+  /// when current branch had activity.
+  @IntProperty(propertyName: 'lastActivity', required: true)
```

Comment:
We should avoid making values required, especially for values that are likely to come from GitHub

Reply:
Sure, in later commits I have a get_branches request handler that relies on the 'lastactivity' timestamp to compare and filter branches later. I will add assertion of not nullable to those